### PR TITLE
Resolve #72 and #78. Added discussion section about the missing features and the SRS evaluation.

### LINKED ISSUE #72: Add discussion sections about G.A.P.
Add discussion sections about the GREAT AFSTAND PROBLEM identified.

- LLM
- UX arrow

### LINKED ISSUE #78: Add srs evaluation section
Add srs evaluation section in the discussion chapter

### PR DESCRIPTION
**Pull request by MadsHeilmann**

### CODE REVIEW

File: src/discussion/srs-evaluation.tex [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+They work seamlessly and efficiently and are well integrated into the overall project.
+
+% textidote: ignore begin
+\subsubsection{Applied learning principles}
+% textidote: ignore begin
+
+Though most features are well implemented as mentioned above, there are some missing features.
+Regarding the applied learning principles the project is missing some key features regarding the learning principles
+mentioned in Section~\ref{sec:learning-methods}.
+Key missing features include explanations and interactivity.
```
Comment (SebastianMygind) [inline src/discussion/srs-evaluation.tex:24]: I understand the thought process of introducing the subsection, then the context and then the specific feature, but it makes the sentence read badly.

**Review by SebastianMygind — CHANGES_REQUESTED** [review]
One minor detail. Otherwise it LGTM

File: src/discussion/srs-evaluation.tex [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+As chess as a game is complex, there are too many different positions for it to make sense to hard-code responses for
+different positions.
+To reach the full potential of the idea, the project needs some form of dynamic language model that is able to interpret
+the Stockfish provided feedback, process this information and provide tailored feedback to the user.
+One way this can be implemented is to use proprietary software such as ChatGPT to generate the feedback.
```
Comment (Danar435) [inline src/discussion/srs-evaluation.tex]: I think that there should be a separate subsection in the discussion going over LLM specifically, as there is a lot we can talk about. You can mention it and refrence the subsection in here instead.

**Review by Danar435 — CHANGES_REQUESTED** [review]

File: src/discussion/srs-evaluation.tex (hunk 1 above)
Comment (MadsHeilmann) [reply src/discussion/srs-evaluation.tex:24]: Agree. Will fix

File: src/discussion/srs-evaluation.tex (hunk 2 above)
Comment (MadsHeilmann) [reply src/discussion/srs-evaluation.tex]: Agree. Will be fixed

**Review by Danar435 — APPROVED** [review]
I wasn't expecting you to write the LLM section now, but I'm not complaining lol, LGTM

**Review by SebastianMygind — APPROVED** [review]

**Pull request merged**